resources/views/game/main.blade.php

```diff
@@ -104,17 +104,18 @@
         </div>
         <div class="bottom-box shadow">
             <div class="item-header">
-                <h2>SPEL ID</h2>
+                <h2>SPEL DETAILS</h2>
             </div>
             <div class="item-box" id="id_box">
-                <p>Het ID van het huidige spel is:</p>
-                <h1>{{$id}}</h1>
+                <p>Het hudige spel:</p>
+                <h1>{{$name}}</h1>
```

Comment:
```suggestion
                <h1>Naam: {{$name}}</h1>
```
gewoon als verduidelijking